pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -863,8 +863,12 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
             if (!Player(player)->isAdministrator()) continue;
             if (IsPlayerAdmin(player)) continue;
 
+            
+    
+            if (Player(player)->isDeveloper())
+                format(playerLevel, sizeof(playerLevel), "Developer");      
```

Comment:
Do we really want to have Developers among this list?

It'll probably be unclear to players that they do not have administrator rights and thus can't help them with houses reports and such

Reply:
Ah, but they can report bugs to devs directly?

Reply:
If you want, I can remove it.

Reply:
A developer has no administrative capability so I think adding them to /admins would be missleading.